src/style-spec/reference/v8.json

```diff
@@ -811,7 +811,8 @@
       "function": "interpolated",
       "zoom-function": true,
       "property-function": true,
-      "doc": "Scale factor for icon. 1 is original size, 3 triples the size.",
+      "units": "factor of the original icon size",
+      "doc": "Scales the original size of the icon by the provided factor. The new pixel size of the image will be `original pixel size * 'image-size'`. 1 is the original size, 3 triples the size of the image.",
```

Comment:
In fe46990085f39f72bd55d168d5c21b778818474b, I replaced the `*` with “multiplied by”. The iOS/macOS SDK’s codegen script hard-wraps the documentation. I wasn’t sure whether the `*` could end up at the beginning of a line, where jazzy would interpret it as a bulleted list. Spelling it out should help the text flow in any case.